lib/data_update_scripts/20201015190914_update_article_main_image_path.rb

```diff
@@ -0,0 +1,15 @@
+module DataUpdateScripts
+  class UpdateArticleMainImagePath
+    def run
+      return unless ENV["FOREM_CONTEXT"] == "forem_cloud"
+
+      Article.find_each do |article|
+        next unless article.main_image&.starts_with? "https://res.cloudinary.com/"
+
+        index = article.main_image.index(URL.url)
+        article.main_image = article.main_image[index..].gsub("/images/", "/remoteimages/")
+        article.save
```

Comment:
Rather than `article.save` could we do this as `.update_column` so we don't run so many callbacks?

Possible also only loop through images `.where.not(main_image: [nil,""])` to avoid looping through so many?

Reply:
Yup let me update this. I didn't try to optimize it too much since this is for forem-cloud only.

Reply:
Ah....... Yeah I wasn't thinking about that. Yeah if this is Forem Cloud only then we don't need that optimization.